feeadjuster/feeadjuster.py

```diff
@@ -204,10 +206,27 @@ def feeadjust(plugin: Plugin):
     return msg
 
 
+@plugin.method("feeadjuster_forward_event_switch")
```

Comment:
I would call the RPC method simply `feeadjuster_toggle` and make the method in a way that when `None` is given it toggles the current state.